step-capstone/src/components/TripList.js

```diff
@@ -33,30 +38,64 @@ class TripList extends React.Component{
     .catch(error => {
       console.log("Error Getting Trips")
       console.error(error)
-    });
+    })
+  }
+
+  componentDidMount() {
+    this.loadTrips();
```

Comment:
this is probably unnesscary to have a method where all it does is call another method

Reply:
oh I see why you did this, more explanation later

Reply:
componentDidMount() ensures the order of implementation: which is after things are being rendered, it will load the trips. So I thought it has some meaning there? What do you think?